pkg/kf/commands/apps/integration_test.go

```diff
@@ -75,6 +78,60 @@ func TestIntegration_Push(t *testing.T) {
 	})
 }
 
+// TestIntegration_Push_manifest pushes the manifest app, using a manifest.yml
+// file. The app is identical to the echo app, and this fact is used to also
+// test manifest file environment variables. It finally deletes the app.
+func TestIntegration_Push_manifest(t *testing.T) {
+	t.Parallel()
+	RunKfTest(t, func(ctx context.Context, t *testing.T, kf *Kf) {
+		currentTime := time.Now().UnixNano()
+		appName := fmt.Sprintf("integration-manifest-%d", currentTime)
+		appPath := filepath.Join(RootDir(ctx, t), "samples", "apps", "manifest")
+
+		// Create a manifest file for this test.
+		// This is so the appName is unique
+		manifestBytes, err := ioutil.ReadFile(filepath.Join(appPath, "manifest.yml"))
```

Comment:
nit: what do you think about moving this to a function to distract less from the rest of the test?